unison-cli/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -1477,12 +1477,12 @@ notifyUser dir o = case o of
   PullSuccessful ns dest ->
     pure . P.okCallout $
       P.wrap $
-        "Successfully updated" <> prettyPath' dest <> "from"
+        "✅ Successfully updated" <> prettyPath' dest <> "from"
           <> P.group (prettyReadRemoteNamespace ns <> ".")
   MergeOverEmpty dest ->
     pure . P.okCallout $
       P.wrap $
-        "The destination" <> prettyPath' dest <> "was empty, and was replaced instead of merging."
+        "✅ Successfully pulled into newly created namespace " <> prettyPath' dest
```

Comment:
```suggestion
        "✅ Successfully pulled into newly created namespace" <> P.group (prettyPath' dest <> ".")
```
or w/e

Reply:
Sorry I didn't think to post this with the original comment — but you want `group` to keep them together in case of a linebreak.

Reply:
Is the `group` important? It wasn't there before.

Reply:
Fixed now :)